events/events.go

```diff
@@ -392,11 +392,11 @@ type ItemDrop struct {
 }
 
 // DataTablesParsed signals that the datatables were parsed.
-// You can use the Parser.SendTableParser() after this event to register update notification on entities & properties.
+// You can use the parser.SendTableParser() after this event to register update notification on entities & properties.
```

Comment:
Ooops, this should reference `Parser.ServerClasses()` - my bad, either way, upper-case

```suggestion
// You can use the Parser.SendTableParser() after this event to register update notification on entities & properties.
```